state/operation.go

```diff
@@ -81,6 +81,11 @@ type operationDoc struct {
 	// If not explicitly set, this is derived from the
 	// status of the associated actions.
 	Status ActionStatus `bson:"status"`
+
+	// ExpectedTaskCount is the number of tasks expected to be completed.
+	// It is used internally for mgo asserts and not exposed via the
+	// Operation interface.
+	ExpectedTaskCount int `bson:"expected-count-count"`
```

Comment:
ExpectedTaskCount is a bit confusing in this context. How about renaming it to something like `SpawnedChildTasks`?